Mendelugram/Mendelugram/View/PhotoDescription/PhotoDescriptionTableViewCell.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  PhotoDescriptionTableViewCell.swift
+//  Mendelugram
+//
+//  Created by Pacek on 21/04/2019.
+//
+
+import UIKit
+
+class PhotoDescriptionTableViewCell: UITableViewCell {
+
+    static let reuseIdentifier = String(describing: PhotoDescriptionTableViewCell.self)
+
+    struct Input {
+        let likesCount: Int
+        let username: String
+        let description: String
+    }
+
+    @IBOutlet weak var likesCountLabel: UILabel!
+    @IBOutlet weak var descriptionLabel: UILabel!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+
+    func configure(input: Input) {
```

Comment:
I think that func configure(with input: Input) would be more in align with standard Swift naming.